README.md

```diff
@@ -15,3 +15,74 @@ Accessibility
 
 Evaluation Date: 201X-XX-XX  
 This project conforms to level AA WCAG 2.0 standards as required by the university's accessibility policy. For more information on the policy please visit: [https://ucomm.stanford.edu/policies/accessibility-policy.html](https://ucomm.stanford.edu/policies/accessibility-policy.html).
+
+System Requirements
+---
+
+To build and deploy code, you must have the following installed on your system:
+
+* Node
+* Grunt
+
+
+### Installation
+
+These tools need only to be installed globally, so if you already have them installed on your system, you are ready to use or add to the toolset.
+
+You may refer to the respective links above for installation instructions for each, but below is a brief overview for each as well:
+
+### Node
+
+Visit Node's website to download the [installer](http://nodejs.org/).
+
+### Grunt
+
+The only requirement for Grunt is Grunts CLI(Command Line Utility).
+If you don't already have that installed, install globally, this is not project specific.
+In NPM we do so with the -g flag.
+
+    npm install -g grunt-cli
+
+### Troubleshooting
+
+* If you have trouble installing any of the tools, you may need to use the sudo command.
+* If you run into any issues with some of the tasks run, npm rebuild and try again.
+`npm rebuild`
+* Also make sure that git is installed globally as some bower packages require it to be fetched and installed.
+
+Development Workflow
+---
+
+### One-time Setup
+
+The following steps need to be performed once to get your local environment up and running.
+
+* Install Docker. Go to [https://www.docker.com/community-edition](https://www.docker.com/community-edition) and download the appropriate file for your system, then install it in your applications folder.
+* Install Lando. Go to [https://github.com/lando/lando/releases](https://github.com/lando/lando/releases) and download the appropriate file for your system, then install it in your applications folder.
+* `cd` to the local directory where you keep your git repos
+* `git clone https://github.com/SU-SWS/decanter.git`
+* `cd decanter`
+* `npm install -g kss`
+* `npm install` to install your remaining dependencies
+
+Workflow
+---
+
+### Starting Your Environment
+
+* Start Docker by double clicking the app icon.
+* Start Lando by double clicking the app icon.
+* Initialize Lando by typing `lando start`. For full Lando documention, see [https://docs.devwithlando.io/](https://docs.devwithlando.io/).
+* To build the styleguide, use `grunt styleguide`.
+* To compile the files, start the watch task, and automatically refresh the browser when changes are made, use `grunt`.
+
+### Development
```

Comment:
We have a wiki page for this. I like having the short notes here but we should add a link to the wiki page.

Reply:
I do think that having some basic, outline-like steps here is still useful and would allow for more advanced users to see what they need and proceed without having to navigate to the Wiki. That said, cleaning up the steps to make them even shorter would be a good idea (e.g., "Start Docker by double clicking the app icon." to "Start Docker.").